README.md

```diff
@@ -1,15 +1,11 @@
 # geckoboard-node
 
-This module requires Node v4 or greater.
-
 > **For users of the deprecated custom widget library**
 
```

Comment:
if you add the `>` markers on the blank lines I think npm and github's README rendering will display it as one box